docs/pages/apis/graphql.fr.mdx

```diff
@@ -0,0 +1,51 @@
+import { Callout } from "nextra-theme-docs";
+
+<Callout type="warning" emoji="⚠️">
+  L'API GraphQL est toujours en travaux et quelques fonctionnalités comme le filtre et le tri ne sont **pas encore** implémentées,  pour le moment ! Des changements sont en attente.
+</Callout>
+
+# API GraphQL
+
+Notre API GraphQL fourni une façon puissante et flexible d'intéragir avec la base de données Potter. Cette section vous présentera les principaux aspects de notre API GraphQL ainsi que la manière de tirer parti de ses capacités.
+
+## Endpoint GraphQL 
+
+Pour effectuer des requêtes et des mutations GraphQL, envoyez vos requêtes `POST` vers l'endpoint GraphQL (`/graphql`) en plaçant vos recherches et variables dans le corps de la requête. L’API attend à ce que le corps de la requête (body) soit au format JSON.
+
+
+## Requêter des données
+
+GraphQL vous permet de demander précisément les données dont vous avez besoin. Pour construire une requête, spécifiez les champs que vous souhaitez récupérer ainsi que leurs relations. Par exemple :
```

Comment:
```suggestion
GraphQL vous permet de demander précisément les données dont vous avez besoin. Pour construire une requête, spécifiez les champs que vous souhaitez récupérer ainsi que leurs relations. Par exemple:
```

Reply:
In French, there is a space before and after a ":"  (like other punctuation marks).
Do you want me to apply your suggestion or leave it the "french way" ? No problem for me if you prefer your suggestion, I'll apply it

Reply:
See reply below